packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -378,7 +380,7 @@ contract LockedGold is
     address communityFund = registry.getAddressForOrDie(GOVERNANCE_REGISTRY_ID);
     address payable communityFundPayable = address(uint160(communityFund));
     require(maxSlash.sub(reward) <= address(this).balance, "Inconsistent balance");
-    communityFundPayable.transfer(maxSlash.sub(reward));
+    communityFundPayable.sendValue(maxSlash.sub(reward));
```

Comment:
this looks like reentrancy is possible. We could add the modifier? I'm unfamiliar with why it may have been omitted here though... @yorhodes

Reply:
wondering if reentrancy in general brings enough benefits to the table to makeup for all the pain it causes ^_^

Reply:
ohhh actually, we're sending to `GOVERNANCE_REGISTRY_ID` here which is our own contract. we could add nonReentrant for peace of mind though?

Reply:
There's no storage changes after the transfer happens. Where are you seeing the possibility of reentrancy?

Reply:
you're right!